ci/Publish-MailDesktopArtifacts.Jenkinsfile

```diff
@@ -0,0 +1,177 @@
+                            if (!fileExists("${desktopWin}")) {
+                                currentBuild.result = 'ABORTED'
+                                error("Unable to find file ${desktopWin}")
+                            }
+
+                            util.downloadFromNexus(	groupId: "app",
+                                                    artifactId: "desktop-mac",
+                                                    version: "${VERSION}",
+                                                    outFile: "${WORKSPACE}/${desktopMac}",
+                                                    fileExtension: 'dmg')
+                            if (!fileExists("${desktopMac}")) {
+                                currentBuild.result = 'ABORTED'
+                                error("Unable to find file ${desktopMac}")
+                            }
+
+                            util.downloadFromNexus(	groupId: "app",
+                                                    artifactId: "desktop-linux",
+                                                    version: "${VERSION}",
+                                                    outFile: "${WORKSPACE}/${desktopLinux}",
+                                                    fileExtension: 'AppImage')
+                            if (!fileExists("${desktopLinux}")) {
+                                currentBuild.result = 'ABORTED'
+                                error("Unable to find file ${desktopLinux}")
+                            }
+
+                            writeFile file: "notes.txt", text: params.releaseNotes
+                            catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS', message: 'Failed to create github release page for desktop') {
+                                withCredentials([string(credentialsId: 'github-access-token', variable: 'GITHUB_TOKEN')]) {
+                                    sh """node buildSrc/createReleaseDraft.js --name '${VERSION} (Desktop)' \
+                                                                           --tag 'tutanota-desktop-release-${VERSION}' \
+                                                                           --uploadFile '${WORKSPACE}/${desktopLinux}' \
+                                                                           --uploadFile '${WORKSPACE}/${desktopWin}' \
+                                                                           --uploadFile '${WORKSPACE}/${desktopMac}' \
+                                                                           --notes notes.txt"""
+                                } // withCredentials
+                            } // catchError
+                            sh "rm notes.txt"
+
+                            stash includes: desktopLinux, name: 'linux_image'
+                        } // script release draft
+                    } // steps
+                } // publish
+                stage('Build webapp') {
```

Comment:
it seems like building web app is a bit out of place here. Would it make sense to build some kind of tarball in its own job?